assignments/this.js

```diff
@@ -1,26 +1,51 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Global Biding. In this instance the 'this' will refer to whatever window you are using. 
```

Comment:
Specifically now just `window`, but whatever the global object is. In browsers, it is indeed `window`, but JavaScript can be run in other environments. In those, the global object might be something else.